elastalert/alerters/thehive.py

```diff
@@ -102,19 +102,21 @@ def alert(self, matches):
             'title': self.create_title(matches),
         }
         alert_config.update(self.rule.get('hive_alert_config', {}))
-        alert_config['description']=self.load_description(alert_config['description'], matches[0])
-        # Iterate through each match found, populating the alert tags and observables as required
-        tags = set()
-        artifacts = []
-        for match in matches:
-            artifacts = artifacts + self.load_observable_artifacts(match)
-            tags.update(self.load_tags(alert_config['tags'], match))
-
-        alert_config['artifacts'] = artifacts
-        alert_config['tags'] = list(tags)
-
-        # Populate the customFields
-        alert_config['customFields'] = self.load_custom_fields(alert_config['customFields'],
+        if not matches:
```

Comment:
This seems backwards. Perhaps change to `if len(matches) > 0:` to make it more clear.

Also, I'm not sure if thehive requires the `artifacts` and `tags` attributes to be present, even if they're empty. It might be safer to make lines 107 and 119 be the only lines inside the `if` check block.

Reply:
I made the change